modules/core/src/v2/coins/cspr.ts

```diff
@@ -201,4 +222,63 @@ export class Cspr extends BaseCoin {
       .call(this)
       .asCallback(callback);
   }
+
+  /**
+   * Explain a Casper transaction from Raw Tx
+   *
+   * @param params
+   * @param callback
+   */
+  explainTransaction(
+    params: ExplainTransactionOptions,
+    callback?: NodeCallback<TransactionExplanation>
+  ): Bluebird<TransactionExplanation> {
+    const self = this;
+    return co<TransactionExplanation>(function*() {
+      const txHex = params.txHex || (params.halfSigned && params.halfSigned.txHex);
+      if (!txHex || !params.fee) {
+        throw new Error('missing explain tx parameters');
```

Comment:
Nit: Missing

Reply:
please see the comment https://github.com/BitGo/BitGoJS/pull/1066#discussion_r596169759. I think it's ok setting the message in lowercase